cmd/serve.go

```diff
@@ -58,18 +59,20 @@ var serveCmd = &cobra.Command{
 		// argument validation
 		validate()
 		log.SetLevel(logLevel)
+		cacheCfg := cache.NewConfig(0 * time.Second)
```

Comment:
```suggestion
		cacheCfg := cache.NewConfig(0 * time.Second)
```
Maybe there should be an enhancement later to configure this cache timeout?

Reply:
This is a global config, individual cache TTLs are configured below